### DISCUSSION TITLE
Images

### DISCUSSION THREAD

**Discussion by aetkie** [UNC-Project-Open-AAC/OS-DPI#113] [Help]
Couple of questions about image use in OS-DPI...

1. What formats are supported? PNG, JPEG? SVG?
2. Does the file name of the image have to match exactly to the label in Excel sheet?
3. Is there a way to download the ARASAAC file as a complete folder or do we download each image individually to our machines?

**Comment by gbishop** [comment]
1. Any format the browser supports should work. All of those for sure.
2. Yes.
3. I think Lori or Sofia fetched them from somewhere online.

**Reply by aetkie** [reply]
@gbishop Thanks for your quick response!

**Comment by sofia-bg** [comment]
I haven't used ARASAAC I have been using https://www.opensymbols.org/ I do have to download individually but I just make a folder on my desktop to download them to and when you upload to OS-DPI you can select all and do all at once. I tried uploading the symbols that we used for the practice layout here, but it uploaded them each individually and then it seemed like to retrieve them you would have to download each individually. I can see if github support has any solutions for uploading multiple files that can then be downloaded at once. In the meantime the symbol files for the current layouts are [here](https://drive.google.com/drive/folders/1-Q4ePrMaP3yPlyJ-pyjP78DO4T3U0V2m?usp=sharing). 

Also you can upload more symbols than you declare in your spreadsheet which can be helpful. As I download symbols I just keep them all in same folder and then when make new layout upload them all even if they aren't all relevant to the data in the spreadsheet for the new layout. That being said I should spend some time to make sure the file I shared is the most recent. I as I said version control is not my greatest strength ;)

**Reply by aetkie** [reply]
@sofia-bg thanks! I was also looking around on opensymbols.org. I started downloading a bunch (the Mulberry symbols). But they were not uploading to OS-DPI for me. I am going to keep trying.

**Reply by sofia-bg** [reply]
Did you name the symbols with the extension in the spreadsheet? So needs to be go.png for example to pull the go symbol in my folder

**Reply by aetkie** [reply]
Good tip @sofia-bg . I did re-name the symbols with the extension. Thy symbols I am using are .svg. So that's reflected in the spreadsheet now. I still can't get the images to load.

**Comment by gbishop** [comment]
You can get everything in the repo at once. It would be nice if you could
get any small part but that isn't one of the choices.

On the front page of the repo there is a green button labeled "Code". Click
it and choose Download Zip. You'll get a HUGE zip file. Open it and then
extract the folder you want.

**Reply by sofia-bg** [reply]
ok will give it a go later thanks

**Comment by gbishop** [comment]
svg's should just work. When you way "not uploading" what do you mean? They
don't show up in the list of media on the content page?

**Reply by aetkie** [reply]
the images show up on the list of media on the content page but not in the  display (is that the right word to use here?)

<img width="610" alt="Screen Shot 2022-10-14 at 2 01 04 PM" src="https://user-images.githubusercontent.com/113473933/195911732-fb5b1866-02f4-4d39-9cea-2f61cc8909d0.png">

**Comment by sofia-bg** [comment]
@aetkie for ARASAAC it looks like if you login and tag things as favorites then you can go to favorites and make a folder. You can drag and drop what your favorites where to the folder. Then if you hover over the folder three dots appear in upper right hand corner and if you select that you have the option to download the whole folder.

**Comment by gbishop** [comment]
Please export your design and email the osdpi file to me.

**Comment by aetkie** [comment]
It's very rough. All I was trying to do was edit the 60 excel sheet to new names and new images to see if I could actually do it.


________________________________

**Reply by sofia-bg** [reply]
@aetkie It has been rough for all of us the first few times. If you export the file you can upload (or email it to us and I can upload here) and share and it will help us to help you figure out why they aren't loading.

**Reply by aetkie** [reply]
Whoops. I replied to the github email instead of composing a new one. I'm just displaying my mistakes here for all to see. :)
